mesa/space.py

```diff
@@ -419,8 +419,12 @@ def is_cell_empty(self, pos: Coordinate) -> bool:
         x, y = pos
         return self.grid[x][y] == self.default_val()
 
-    def move_to_empty(self, agent: Agent, cutoff: float = 0.998) -> None:
+    def move_to_empty(
+        self, agent: Agent, cutoff: float = 0.998, num_agents: int | None = None
+    ) -> None:
         """Moves agent to a random empty cell, vacating agent's old cell."""
+        if num_agents is not None:
+            warn("num_agents is being deprecated since it's no more necessary", DeprecationWarning)
```

Comment:
Should be "no longer necessary" instead.

Reply:
done